craft_parts/packages/errors.py

```diff
@@ -26,6 +26,16 @@ class PackagesError(PartsError):
     """Base class for package handler errors."""
 
 
+class PackageBackendNotSupported(PartsError):
+    """Requested package resolved not supported on this host."""
+
+    def __init__(self, backend: str) -> None:
+        self.backend = backend
+        super().__init__(
+            brief=f"Package backed {backend!r} is not supported on this environment.",
```

Comment:
s/backed/backend/

Reply:
fixed in 65ca1ee2bf780ef7a31e10e01738c14c34ae9d40